courses-app/src/app/shared/pipes/creation-date/creation-date.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
do you really need this pipe? why not just use build-in

Reply:
it was a part of task